macros/edr/dbt_artifacts/upload_artifacts_to_table.sql

```diff
@@ -11,7 +11,7 @@
 
     {% if append %}
         {# In append mode, just insert, and no need to be atomic #}
-        {% do elementary.insert_rows(table_relation, flatten_artifact_dicts, should_commit, elementary.get_config_var('dbt_artifacts_chunk_size')) %}
+        {% do elementary.insert_rows(table_relation, flatten_artifact_dicts, should_commit, elementary.get_config_var('dbt_artifacts_chunk_size'), on_query_exceed) %}
```

Comment:
Note that I haven't drilled it down to `replace_table_data` and `delete_and_insert` for now, but I think that's fine for the time being.